history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/SimpleHistoryServiceImpl.java

```diff
@@ -59,4 +59,13 @@ public void create(TaskanaHistoryEvent event) {
   public HistoryQuery createHistoryQuery() {
     return new HistoryQueryImpl(taskanaHistoryEngine, historyQueryMapper);
   }
+
+  /*
+   * ATTENTION: This method exists for testing purposes.
+   */
+  @VisibleForTesting
```

Comment:
This was a stupid suggestion - Sorry. Please remove together with the guava dependency. Thanks!